bgcval2/functions/globalVolMean.py

```diff
@@ -0,0 +1,109 @@
+#
+# Copyright 2015, Plymouth Marine Laboratory
+#
+# This file is part of the bgc-val library.
+#
+# bgc-val is free software: you can redistribute it and/or modify it
+# under the terms of the Revised Berkeley Software Distribution (BSD) 3-clause license.
+
+# bgc-val is distributed in the hope that it will be useful, but
+# without any warranty; without even the implied warranty of merchantability
+# or fitness for a particular purpose. See the revised BSD license for more details.
+# You should have received a copy of the revised BSD license along with bgc-val.
+# If not, see <http://opensource.org/licenses/BSD-3-Clause>.
+#
+# Address:
+# Plymouth Marine Laboratory
+# Prospect Place, The Hoe
+# Plymouth, PL1 3DH, UK
+#
+# Email:
+# ledm@pml.ac.uk
+#
+"""
+.. module:: globalVolMean 
+   :platform: Unix
+   :synopsis: This function calculates the volume weighted mean of a field.
+
+.. moduleauthor:: Lee de Mora <ledm@pml.ac.uk>
+
+"""
+
+import numpy as np
+from ..bgcvaltools.dataset import dataset
+import os, sys
+import errno
+
+#from bgcvaltools.bgcvalpython import Area
+
+tmask     = 0
+pvol     = 0
+loadedArea = False
+
+def loadDataMask(gridfn):
+    """
+    Load data mask, heopfully only once!
+    """
+    global loadedArea
+    global tmask
+    global pvol
+    if not gridfn or not os.path.exists(gridfn):
+        raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), gridfn)
+
+    nc = dataset(gridfn, 'r')        
+    try:
+        pvol   = nc.variables['pvol'][:]
+        tmask = nc.variables['tmask'][:]
+    except:
```

Comment:
can one var raise a `KeyError` coz it's not in the file, and another not, since it is? Would you use the one that doesn't raise (that exists in the file, that is)? coz excepting after both means you discard them both